go.mod

```diff
@@ -21,9 +21,11 @@ require (
 	github.com/lestrrat-go/jwx v1.2.25
 	github.com/nats-io/nats.go v1.16.0
 	github.com/panjf2000/ants/v2 v2.5.0
+	github.com/pion/dtls/v2 v2.1.6-0.20220815090156-a04cfccb23e5
+	github.com/pion/logging v0.2.2
 	github.com/plgd-dev/device/v2 v2.0.0-20220929115241-033fe063adb5
 	github.com/plgd-dev/go-coap/v2 v2.6.1-0.20220801124204-0ab75f6e0680
```

Comment:
Shouldn't this branch only have `go-coap/v3` instaed of both v2 and v3?

Reply:
Fixed